### DISCUSSION TITLE
Cannot find module 'vue-data-ui' or its corresponding type declarations. ts-plugin(2307)

### DISCUSSION THREAD

**Discussion by Davod01** [graphieros/vue-data-ui#22] [Q&A] [answered]
hello,
i using quesar framework and when i import vue-data-ui i getting this bellow error
![Untitled](https://github.com/graphieros/vue-data-ui/assets/32578582/a9ec4c66-37d6-4119-acad-a268b78fad2e)
how can i import this library type declarations

**Comment by graphieros** [comment, marked as answer]
Hey :)

You can upgrade to `v2.1.38` or latest which solves this problem without having to copy the .d.ts file.

With prior versions you need to copy paste into your project the vue-data-ui.d.ts file, located in the package inside the types folder, which contains all the type declarations.

Cheers

**Reply by Davod01** [reply]
thank you and thank you again for this amazing library

**Reply by graphieros** [reply]
Thank you very much!
